sdk/identity/identity/src/constants.ts

```diff
@@ -17,3 +17,13 @@ export const DeveloperSignOnClientId = "04b07795-8ddb-461a-bbee-02f9e1bf7b46";
  * @ignore
  */
 export const DefaultTenantId = "common";
+
+/**
+ * A list of known authority hosts
+ */
+export enum KnownAuthorityHosts {
+  AzureChina = "login.chinacloudapi.cn",
```

Comment:
Can you verify that you don't need https://?

Reply:
it looks like this does need https:// as well from looking at how it's handled in `ClientSecretCredential` at least.

https://github.com/Azure/azure-sdk-for-js/blob/14c1a6f01010ecb4b0a9838ab967fc6ccecdd5ba/sdk/identity/identity/src/credentials/clientSecretCredential.ts#L63-L64

Reply:
Updated